components/NavBar.tsx

```diff
@@ -0,0 +1,32 @@
+// CSS
+import styles from '../styles/components/NavBar.module.scss';
+// Config
+import { pageInfo } from '../config.yaml';
+// Component
+interface Props {
+  currentPage: string;
+}
+export default function NavBar(props: Props) {
+  // Build The Nav
+  const navContent: JSX.Element[] = [];
+  pageInfo.forEach((page, index) => {
```

Comment:
Just because it just came up at work, is index our best option here or is there another ID we could use? I guess with the config.yaml there wouldn't be any opportunities for these to get moved around without us knowing

Reply:
Bassically from my understanding of react the key matters more for virtual Dom updates this shouldn't ever update in the virtual Dom and rebuilds are irrelevant